package.json

```diff
@@ -57,9 +57,10 @@
     "nps-utils": "^1.1.2",
     "opt-cli": "^1.5.1",
     "prettier-eslint-cli": "^3.1.2",
-    "react": "^15.4.2",
-    "react-addons-test-utils": "^15.4.2",
-    "react-dom": "^15.4.2",
+    "prop-types": "^15.5.1",
+    "react": "^15.5.1",
+    "react-addons-test-utils": "^15.5.0",
```

Comment:
This is still required until this gets merged: https://github.com/airbnb/enzyme/pull/876